src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -216,7 +216,8 @@ void regclass_Tensor(py::module m) {
 
     cls.def(py::init([](py::object& image) {
                 if (!py::isinstance(image, py::module::import("PIL.Image").attr("Image"))) {
-                    throw py::type_error("Input must be a PIL.Image.Image object");
+                    throw py::type_error(
+                        "Input argument must be a PIL.Image.Image/numpy.array/List[int, float, str] object");
```

Comment:
python scalars?

Reply:
even if scalar, it should be wrapped to numpy.array